README.md

```diff
@@ -54,20 +110,25 @@ pgdbname=dcrdex_testnet
 pgpass=dexpass
 ```
 
-The **app data directory** (*~/.dcrdex/*) is the default location used by the
-DEX server, but can be customized with a command-line argument.
+The *~/.dcrdex/* is the default **app data directory** location used by the
+DEX server, but can be customized with the `--appdata` command-line argument.
 
 Master public keys are network-specific, so **dcrd** and **dcrwallet** will need
 to be run on the same network on which the server will be run.
 
-**Run your asset daemons**. As of writing, only `dcrd`, `bitcoind`, and
-`litecoind` are supported. The `txindex` configuration option must be set.
+#### Run your asset daemons.
+
+As of writing, only `dcrd`, `bitcoind`, and `litecoind` are supported. The
+`txindex` configuration option must be set.
+
+#### Create the asset and market configuration file
```

Comment:
Forgive the nitpicking
```suggestion
#### Create the assets and markets configuration file
```

Reply:
I'm not sure about this. I don't think I would call it an "assets configuration file", I think I would call it an "asset configuration file", even if it configures more than one asset. Same with markets.

Reply:
> Forgive the nitpicking

Forgive? I encourage it!